atomic_reactor/schemas/config.json

```diff
@@ -627,6 +615,17 @@
             "type": "string"
         }
     }
```

Comment:
We are mising a ',' here and another one after `description`.

Reply:
Thanks, fixed.